kernel/packages/shared/renderer/sagas.ts

```diff
@@ -97,11 +97,6 @@ function* handleMessageFromEngine(
   const { type, jsonEncodedMessage } = action.payload
   DEBUG && logger.info(`handleMessageFromEngine`, action.payload)
   if (_instancedJS) {
-    if (type === 'PerformanceReport') {
-      _instancedJS.then(($) => $.onMessage(type, jsonEncodedMessage)).catch((e) => logger.error(e.message))
-      return
-    }
```

Comment:
@BrianAmadori @eordano this was completely breaking the PerformanceReporting, it never worked with the since the "samples" were added

Reply:
Not sure :/

PerformanceReport is not JSON-encoded to avoid one extra encoding -- it comes as a string of chars